services/voice_service/requirements.txt

```diff
@@ -0,0 +1,18 @@
+click==8.0
+nltk==3.5
+flask==2.2.2
+itsdangerous==2.0.1
+gunicorn==19.9.0
+requests==2.22.0
+sentry-sdk[flask]==0.14.1
+healthcheck==1.3.3
+jinja2<=3.0.3
+Werkzeug==2.2.2
+df_engine==0.8.1
+six==1.15.0
+pandas>=1.3.5
+SoundFile==0.10.3.post1
+fire>=0.5.0
+kaldiio>=2.17.2
+matplotlib>=3.5.3
+PyYAML>=6.0
```

Comment:
ity is better not to use >= please.

Reply:
`==` -> `>=` done

Reply:
no, I meant exactly opposite. 
Better NOT to use `>=`